consul/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
```

Comment:
That's a good question. I think yea, that we just don't know yet, and we're also not at version 6 yet, so we don't wanna say we'll keep this exactly compatible until it's finalized.

This is all from the skeleton, though:

https://github.com/DataDog/datadog-sdk-testing/blob/master/lib/config/README.md